src/metabase/driver/teradata.clj

```diff
@@ -0,0 +1,306 @@
+(ns metabase.driver.teradata
+  (:require [metabase.driver :as driver]
+    [clojure.java.jdbc :as jdbc]
+    [clojure [set :refer [rename-keys], :as set]
+     [string :as s]]
+    [honeysql [core :as hsql]
+     [format :as hformat]
+     [helpers :as h]
+     types]
+    [metabase.models [field :as field]]
+    [clojure.tools.logging :as log]
```

Comment:
We try to keep all our requires sorted. If you're using emacs you can have it do it for you automatically with the `cljr-clean-ns` command from the `clj-refactor` package (just make sure you `(setq cljr-favor-prefix-notation t)`). Otherwise take a look at other namespaces in the project for an example. We just sort them alphabetically, e.g. `clojure.tools.logging` should come before `metabase.models.field`.